Compiler/constants.js

```diff
@@ -2,7 +2,7 @@ let ActiveLanguage = localStorage.getItem("ActiveLangugae");
 //
 
 if (ActiveLanguage == null) {
-  ActiveLanguage = "Hindi";
```

Comment:
this is for default language that is Hindi if none is selected. Will have to remove this change.

Reply:
sure